shakedown/dcos/package.py

```diff
@@ -100,7 +106,7 @@ def install_package(
         future = now + timeout_sec
 
         while now < future:
-            if shakedown.get_service(package_name):
+            if shakedown.get_service(app_id):
```

Comment:
I don't know why we use app_id at all.   From the perspective of the end user they have zero control over app_id.   package_name 'marathon' and service.name 'marathon-user' as the default is what they have control over.

If we launch multiple marathons, they will need 1) different app_id and 2) different service_names.   however by convention in dcos they are the same.   can we call this service name?